src/Training.AirBnb.Clone.Backend/AirBnB.Api/appsettings.json

```diff
@@ -5,5 +5,6 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
-}
+  "AllowedHosts": "*",
+  "AirBnBNotificationsDatabaseConnection": "Host=localhost;Port=5433;Database=TrainningAirBnB;Username=postgres;Password=postgres"
```

Comment:
Add connection strings inside "ConnectionStrings" section and move connection strings to appsettings.Development.json